pysnark/boolean.py

```diff
@@ -0,0 +1,186 @@
+import pysnark.runtime
+from pysnark.runtime import LinComb, ConstVal, PrivVal, PubVal, add_constraint
+
+"""
+Support for Boolean values
+Costs 0 constraints to convert a LinComb to a LinCombBool
+"""
+
+class LinCombBool:
+    def __init__(self, lc):
+        """
+        Constructs a LinCombBool
+        Costs 2 constraints

```

Comment:
This costs just one constraint, right?

Reply:
Yes, just one. I updated the comments in a future commit.